### PR TITLE
Damienbod/aspnet/core/security/authorization/resourcebased

### PR DESCRIPTION
**Pull request by damienbod**
Update to .NET Core 3.0

https://docs.microsoft.com/en-us/aspnet/core/security/authorization/resourcebased

support for 1.1 and 2.2, 3.0

https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/authorization/resourcebased.md

### CODE REVIEW

File: aspnetcore/security/authorization/resourcebased.md [hunk 1]
```diff
@@ -60,13 +70,13 @@ In the following example, the resource to be secured is loaded into a custom `Do
 
 ::: moniker range=">= aspnetcore-2.0"
 
-[!code-csharp[](resourcebased/samples/ResourceBasedAuthApp2/Pages/Document/Edit.cshtml.cs?name=snippet_DocumentEditHandler)]
+[!code-csharp[](resourcebased/samples/3_0/ResourceBasedAuthApp2/Pages/Document/Edit.cshtml.cs?name=snippet_DocumentEditHandler)]
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/resourcebased.md]: The 3.0 folder doesn't have Pages. This path is incorrect (see the build report from the details tab)

File: aspnetcore/security/authorization/resourcebased.md [hunk 2]
```diff
@@ -127,15 +148,15 @@ To call an operational resource handler, specify the operation when invoking `Au
 
 ::: moniker range=">= aspnetcore-2.0"
 
-[!code-csharp[](resourcebased/samples/ResourceBasedAuthApp2/Pages/Document/View.cshtml.cs?name=snippet_DocumentViewHandler&highlight=10-11)]
+[!code-csharp[](resourcebased/samples/3_0/ResourceBasedAuthApp2/Pages/Document/View.cshtml.cs?name=snippet_DocumentViewHandler&highlight=10-11)]
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/resourcebased.md]: bad path here  too

File: aspnetcore/security/authorization/resourcebased.md (hunk 1 above)
Comment (damienbod) [reply aspnetcore/security/authorization/resourcebased.md]: fixed, thanks

File: aspnetcore/security/authorization/resourcebased.md (hunk 2 above)
Comment (damienbod) [reply aspnetcore/security/authorization/resourcebased.md]: fixed, thanks

**Comment by Rick-Anderson** [comment]
@damienbod  do a `git pull` before submitting any more commit. I had to fix `== aspnetcore-2.0` again. Also take a look at the details link for errors and warnings.

**Comment by Rick-Anderson** [comment]
@damienbod  thanks!
Check my commit for how I fixed monikers

I used 
 `::: moniker range=">= aspnetcore-2.0 < aspnetcore-3.0"`

**Pull request merged**